tests/homePage.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from '@playwright/test';
```

Comment:
As mentionned in our earlier discussion, I rather like an iterative approach. Therefore, would you mind providing the tests in a separate PR? It would be nice to focus first, on the sole setup of Playwright.